src/__tests__/fightClass.spec.ts

```diff
@@ -1,14 +1,16 @@
 import { Fight } from "../fightClass"
-
+import { PokemonType } from "../pokemonClass"
+import { PokemonFactory } from "../pokemonFactory"
+import * as data from "../../assets/poke_data.json"
```

Comment:
What I am afraid of is that: If we change `poke_data` for balance, then **tests will fail**. Any change in attack, hp or something of rattata, abra or any pokemon used there will result in change of expected numbers.
What I suggest is using constructor for pokemons: `new Pokemon('Poke1', '', 50, 50, 50)` - or something